TwitchPlaysAssembly/Src/Commands/GlobalCommands.cs

```diff
@@ -91,6 +91,59 @@ public static void ResetUser([Group(1)] string parameters, string user, bool isW
 		}
 	}
 
+	#region Voting
+	/// <name>Start a vote</name>
+	/// <syntax>vote [action]</syntax>
+	/// <summary>Starts a vote about doing an action</summary>
+	[Command(@"vote (togglevs)")]
+	public static void VoteStart(string user, [Group(1)] bool VSMode) => Votes.StartVote(null, user, VSMode ? "VS Mode Toggle" : "");
```

Comment:
```suggestion
	public static void VoteStart(string user) => Votes.StartVote(null, user, "VS Mode Toggle");
```
The `VSMode` variable will never be false since that group is required.

Reply:
Same as the Detonation one